spec/datadog/tracing/contrib/sinatra/activerecord_spec.rb

```diff
@@ -1,122 +0,0 @@
-# typed: ignore
```

Comment:
Why was this test file removed? Does it not provide value? Why was it added in first place?

Reply:
I think this test is duplicated with our ActiveRecord test, and does not provide much value.